cyences_app_for_splunk/bin/user_inventory_v2_util.py

```diff
@@ -378,6 +392,9 @@ def add_user_entry(self, new_entry: UserEntry):
         # Assign a unique user_uuid to the user
         matching_user = self._find_user(new_entry)
 
+        if matching_user is None:
```

Comment:
Logically this looks not right.

Use the functionis_user_ends_with separately and directly here.